### PR TITLE
Fix Windows environment set issue and change to env to params for compatibility

### PR DESCRIPTION
**Pull request by sinankeskin**
Hi.

All environment values changed to -- params for compatibility reasons.

I've tested on my project (on Windows) and working as expected. ( Except --url param which I don't have but I assume it'll work.)

Thank you for this great addon.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,22 +64,26 @@ module.exports = {
...
-          url ? `SENTRY_URL=${url} ` : '' +
-          `SENTRY_ORG=${orgName} ` +
-          `SENTRY_PROJECT=${appName} ` +
-          `SENTRY_AUTH_TOKEN=${authToken} ` +
-          `node_modules/.bin/sentry-cli ${command}`
+          [
+            path.join('node_modules', '.bin', 'sentry-cli'),
+            url ? `--url ${url}` : '',
+            `--auth-token ${authToken}`,
+            'releases',
```
Comment (Exelord) [inline index.js:82]: I preferred to have `releases` in every command. `sentryCliExec` is meant to execute sentry cli not specific releases commands. :) Could you please revert that?

File: index.js [hunk 2]
```diff
@@ -64,22 +64,26 @@ module.exports = {
         const releaseName = `${appName}@${this.readConfig('revisionKey')}`;
 
         this.log('SENTRY: Deleting release...');
-        this.sentryCliExec(`releases delete ${releaseName}`);
+        this.sentryCliExec(`delete ${releaseName}`);
         this.log('SENTRY: Release deleted!');
       },
 
-      sentryCliExec(command) {
+      sentryCliExec(params) {
```
Comment (Exelord) [inline index.js:71]: We should get back to command :/ Params would an array of object of options.

File: index.js (hunk 1 above)
Comment (sinankeskin) [reply index.js:82]: With -- parameters you can't do that.

sentry-cli has a certain order rule for that. --org and --project parameters must always come after command.

At first I was going to write like this. this.sentryCliExec('release', `new ${releaseName}`);

Two arguments first command, second params. That's why it left named params.

File: index.js (hunk 2 above)
Comment (sinankeskin) [reply index.js:71]: If this.sentryCliExec('release', `new ${releaseName}`); style ok to you. I'll change the signature with as sentryCliExec(command, params)?

**Comment by sinankeskin** [comment]
How about now? I think this is better.

**Pull request merged**